lib/Partitioner/Partitioner.cpp

```diff
@@ -425,6 +426,39 @@ Expected<DAGListTy> Partitioner::createDAGWithoutPartition(
   }
 
   NodeToFunctionMap mapping;
+  for (auto func : module_->getFunctions()) {
+    mapping.createPartition(func, backendName);
+
+    GraphMemInfo graphMem;
+    graphMem.constMemSize = func->getParent()->getConstantsSize();
```

Comment:
The parent of function is the Module, this will report the constant size for the whole module,
To get just the constants used by the function you can use func->findConstants which can then be summed.

Reply:
Good catch. Made the appropriate change in latest push.